### PR TITLE
docs: improve developer and public project documentation

### PR DESCRIPTION
**Pull request by wreality**
Updates to the Contributing section of the docs site.

Preview build is here: https://deploy-preview-112--eloquent-galileo-3be94a.netlify.app/contributing/

### Review Criteria
- [x] General code review
- [x] Language / content review

### CODE REVIEW

File: docs/contributing/testing.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+To run the client-side unit tests, you can run the following **from the `/client` directory**:
+
+```sh
+lando yarn test:unit
+```
+Each package's documentation is an excellent source of information on testing best practices and examples.  The [Vue Testing Handbook](https://lmiller1990.github.io/vue-testing-handbook/) is also an excellent resource for unit testing a Vue application.
+### Lando config for Majestic UI
+
+The [Majestic](https://github.com/Raathigesh/majestic) GUI for jest is a useful tool for running unit tests in a browser and watching code and tests for changes.  You can add the following config to your `.lando.local.yml` file to enable a container for Majestic.
```
Comment (gmeben) [inline docs/contributing/testing.md:30]:
```suggestion
The [Majestic](https://github.com/Raathigesh/majestic) GUI for Jest is a useful tool for running unit tests in a browser and watching code and tests for changes.  You can add the following config to your `.lando.local.yml` file to enable a container for Majestic.
```

**Review by gmeben — CHANGES_REQUESTED** [review]
Found some typos and grammatical fixes and suggested some improvements for readability and stylistic changes.

File: docs/contributing/documentation.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Documentation
+
+CCR's documentation (the site you're currently reading) is a [VuePress](https://vuepress.vuejs.org) site located in the `/docs` directory of the repo.  Our goal is to iterate the documentation as features are added to the application.  All of the documentation files are [Markdown](https://www.markdownguide.org/getting-started/) with some minor Vue additions sprinkled in here and there.
```
[the remark this answers is not in the document]
Comment (wreality) [reply docs/contributing/documentation.md:3]: This changes the meaning of what I was going for very slightly, but I'm cool with it.

File: docs/contributing/README.md [hunk 3]
```diff
@@ -1,2 +1,83 @@
...
+All contributors to the project are must agree to adhere to our project's [Code of Conduct](https://github.com/MESH-Research/CCR/blob/master/CODE_OF_CONDUCT.md)
+
+::: tip In summary
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
```
[the remark this answers is not in the document]
Comment (wreality) [reply docs/contributing/README.md]: No newline is good with me.  I think a former version of the markdown lint had a line limit rule of 80 characters.  That was adjusted to 400, but this was probably written before the autoformatter was doing its thing in my VSCode.  Either way, agreed.

**Comment by wreality** [comment]
Thanks much for looking it over in such detail.  With such a big chunk of text, I was sure there were bound to be errors and typos, try as I might.  Changes accepted, ready for approval.

File: docs/contributing/documentation.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (gmeben) [reply docs/contributing/documentation.md]: Ah, then maybe to make it more clear it should've been written as, "Our goal is to iterate the documentation alongside features added to the application."

**Review by gmeben — APPROVED** [review]
Looking very good. Feel free to make further changes as you see fit.

**Pull request merged**